app/src/main/java/com/dimelo/sampleapp/MyFirebaseMessagingService.java

```diff
@@ -1,23 +1,20 @@
 package com.dimelo.sampleapp;
 
-import android.util.Log;
 
 import com.dimelo.dimelosdk.main.Dimelo;
-import com.google.firebase.iid.FirebaseInstanceId;
 import com.google.firebase.messaging.FirebaseMessagingService;
 import com.google.firebase.messaging.RemoteMessage;
 
 public class MyFirebaseMessagingService extends FirebaseMessagingService {
-
     /**
      * Called when message is received.
      *
      * @param remoteMessage Object representing the message received from Firebase Cloud Messaging.
      */
+
     @Override
     public void onMessageReceived(RemoteMessage remoteMessage) {
-        MainActivity.setupDimelo(MyFirebaseMessagingService.this);
-        Log.e("hhhh", remoteMessage.getData() + "");
+        RcConfig.setupDimelo(MyFirebaseMessagingService.this);
         if (Dimelo.consumeReceivedRemoteNotification(MyFirebaseMessagingService.this, remoteMessage.getData(), null)){
```

Comment:
```suggestion

        if (Dimelo.consumeReceivedRemoteNotification(MyFirebaseMessagingService.this, remoteMessage.getData(), null)){
```

Reply:
in this case let's also put a space between the `){`